src/app/components/Overview/components/Tables/Operators/Operators.tsx

```diff
@@ -43,73 +42,51 @@ const Operators = () => {
     SsvNetwork.getInstance().fetchOperators({ page: 1, validatorsCount: 'true' })
       .then((result: any) => {
         overviewStore.setTotalOperators(result.data.pagination.total);
+        if (result.data.operators.length > 0) setOperatorsExist(true);
```

Comment:
For better readability try to avoid placing the code in the same line where condition is

Reply:
done